src/components/Layouts/ReportLayout.less

```diff
@@ -118,7 +129,6 @@ h1 {
       }
 
       .section-item-value {
-        .section-markdown,
```

Comment:
we don't need custom css for markdown anymore? its mostly clearing margins

Reply:
done